Structure_Engine/Create/Loads/BarTemperatureLoad.cs

```diff
@@ -37,7 +37,16 @@ public static partial class Create
         /***************************************************/
         /**** Public Methods                            ****/
         /***************************************************/
-       
+
+
+        [Description("Creates an uniform temprature load to be applied to Bars.")]
```

Comment:
a

Reply:
beaten by exceptions again! https://english.stackexchange.com/questions/4700/is-it-a-uniform-or-an-uniform